pkg/grpc/authn/tokenbased/extractor.go

```diff
@@ -37,7 +37,7 @@ func (e *extractor) IdentityForRequest(ctx context.Context, ri requestinfo.Reque
 	}
 	token, err := e.validator.Validate(ctx, rawToken)
 	if err != nil {
-		logging.GetRateLimitedLogger().WarnL(
+		logging.GetRateLimitedLogger().DebugL(
 			ri.Hostname,
 			"Token validation failed for hostname %v: %v",
 			ri.Hostname,
```

Comment:
@ivan-degtiarenko do you remember why you decided to redefine the error class in https://github.com/stackrox/stackrox/pull/18/files#diff-f96b8bb915687e8420a366a6177912af551be618f8f9fce6802a0d0e85bc2e0aR55-R56 ?